### PR TITLE
Added help button to main menu #5566

### PR DESCRIPTION
**Pull request by Beep6581**
Opens RawPedia in the default URI handler

Works in Linux Sabayon/KDE/Chrome

### CODE REVIEW

File: rtgui/rtwindow.cc [hunk 1]
```diff
@@ -913,6 +919,14 @@ void RTWindow::writeToolExpandedStatus (std::vector<int> &tpOpen)
 }
 
 
+void RTWindow::showRawPedia()
+{
+    const Glib::ustring& uri = "https://rawpedia.rawtherapee.com/";
```
Comment (Floessie) [inline rtgui/rtwindow.cc]: This (accidentally?) makes use of the ["most important `const`"](https://herbsutter.com/2008/01/01/gotw-88-a-candidate-for-the-most-important-const/). :wink:

**Comment by Beep6581** [comment]
> Why not this->show_uri(uri, err);
> 
> ?

For the most important reason that I didn't know any better ;)

**Comment by Floessie** [comment]
@Beep6581 Spoiler alert: if you want to compact it yourself, don't read on... :wink:









How about this:

```patch
diff --git a/rtgui/rtwindow.cc b/rtgui/rtwindow.cc
index 7f3c409a6..785764909 100644
--- a/rtgui/rtwindow.cc
+++ b/rtgui/rtwindow.cc
@@ -918,13 +918,9 @@ void RTWindow::writeToolExpandedStatus (std::vector<int> &tpOpen)
     }
 }
 
-
 void RTWindow::showRawPedia()
 {
-    const Glib::ustring& uri = "https://rawpedia.rawtherapee.com/";
-    guint32 err = 0;
-    RTWindow *rtWin = this;
-    rtWin->show_uri(uri, err);
+    show_uri("https://rawpedia.rawtherapee.com/", GDK_CURRENT_TIME);
 }
 
 void RTWindow::showICCProfileCreator ()
```

**Comment by Beep6581** [comment]
@Floessie :sob: :smile:

**Comment by Beep6581** [comment]
Ready for merge.

**Comment by heckflosse** [comment]
Clicking the help button on Windows7 rawpedia opens fine but I get this in console:
```
$ Release/rawtherapee

(rawtherapee.exe:4704): Gtk-WARNING **: 22:11:40.037: Couldn't export handle, unsupported windowing system
[Child 6824, Chrome_ChildThread] WARNING: pipe error: 109: file z:/task_1575282935/build/src/ipc/chromium/src/chrome/common/ipc_channel_win.cc, line 341
[Child 6824, Chrome_ChildThread] WARNING: pipe error: 109: file z:/task_1575282935/build/src/ipc/chromium/src/chrome/common/ipc_channel_win.cc, line 341
[Child 8072, Chrome_ChildThread] WARNING: pipe error: 109: file z:/task_1575282935/build/src/ipc/chromium/src/chrome/common/ipc_channel_win.cc, line 341
[Child 8072, Chrome_ChildThread] WARNING: pipe error: 109: file z:/task_1575282935/build/src/ipc/chromium/src/chrome/common/ipc_channel_win.cc, line 341
[Child 4676, Chrome_ChildThread] WARNING: pipe error: 109: file z:/task_1575282935/build/src/ipc/chromium/src/chrome/common/ipc_channel_win.cc, line 341
[Child 4676, Chrome_ChildThread] WARNING: pipe error: 109: file z:/task_1575282935/build/src/ipc/chromium/src/chrome/common/ipc_channel_win.cc, line 341
[Parent 4148, Gecko_IOThread] WARNING: pipe error: 109: file z:/task_1575282935/build/src/ipc/chromium/src/chrome/common/ipc_channel_win.cc, line 341
[Child 6008, Chrome_ChildThread] WARNING: pipe error: 109: file z:/task_1575282935/build/src/ipc/chromium/src/chrome/common/ipc_channel_win.cc, line 341
[Child 6008, Chrome_ChildThread][GPU 5440, Chrome_ChildThread] WARNING: pipe error: 109: file z:/task_1575282935/build/src/ipc/chromium/src/chrome/common/ipc_channel_win.cc, line 341

###!!! [Child][RunMessage] Error: Channel closing: too late to send/recv, messages will be lost


###!!! [Child][MessageChannel::SendAndWait] Error: Channel error: cannot send/recv
```

**Pull request merged**